Add Gaomon GM116HD

longnguyen2004 opened the pull request:
Values extrapolated from GM156HD, and confirmed with real tablet.

jamesbt365 reviewed (changes requested):
Please convert to the configuration format seen in master.

jamesbt365 commented on OpenTabletDriver.Configurations/Configurations/Gaomon/GM116HD.json:
```diff
@@ -0,0 +1,43 @@
+  "Specifications": {
+    "Digitizer": {
+      "Width": 257.4,
+      "Height": 143.6,
```
The height value should be 143.4 if the Max Y value below is correct with the calculation MaxY/lpmm (28680/200)

jamesbt365 commented on OpenTabletDriver.Configurations/Configurations/Gaomon/GM116HD.json:
```diff
@@ -0,0 +1,43 @@
+    "Digitizer": {
+      "Width": 257.4,
+      "Height": 143.6,
+      "MaxX": 51480.0,
```
Not sure if you manually guessed these, but these need to be checked in the tablet > tablet debugger if you haven't already.

longnguyen2004 commented:
Updated values (the previous one was taken from a reseller's website and it was slightly off, changed to [official values](https://gaomon.cn/Pen_Display/GM116HD_GM156HD.html#:~:text=11.6inch-,%E5%B7%A5%E4%BD%9C%E5%8C%BA%E5%9F%9F,256.32%20x%20144.18mm,-%E6%98%BE%E7%A4%BA%E5%88%86%E8%BE%A8%E7%8E%87))

jamesbt365 reviewed (commented):
Good to go once a diagnostic is provided with the tablet plugged in and working and the tablet is added to the tablets.md in the root of the directory.

longnguyen2004 commented:
[diagnostics.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/12436088/diagnostics.txt)
Updated the support table

X9VoiD commented:
@InfinityGhost CI is stuck here again, I can't bypass the checks to merge this.
